core/package.json

```diff
@@ -60,6 +60,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-custom-rules": "file:custom-rules",
     "execa": "^5.0.0",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
This is required since we access it directly here: https://github.com/ionic-team/ionic-framework/blob/ee6ba200d159c1380816453ddc17699edc1d4bf1/core/src/utils/test/playwright/matchers/toHaveReceivedEventDetail.ts#L2